.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
+
+on: pull_request
+
```

Comment:
- [ ] You are having failing linters,  Linter result should be green. AKA: Linter should report NO errors.